frontend/lang/messages/en-US.json

```diff
@@ -985,7 +985,8 @@
       "food-data": "Food Data",
       "example-food-singular": "ex: Onion",
       "example-food-plural": "ex: Onions",
-      "label-overwrite-warning": "This will assign the chosen label to all selected foods and potentially overwrite your existing labels."
+      "label-overwrite-warning": "This will assign the chosen label to all selected foods and potentially overwrite your existing labels.",
+      "on-hand-checkbox-label": "Flag this as a food you have 'on hand', to set it as unchecked by default when adding a recipe to a shopping list."
```

Comment:
Maybe something like:

"Setting this flag will make this food unchecked by default when adding a recipe to a shopping list."

(See below comments which include the text "On Hand" already)